toolchain/check/testdata/index/fail_negative_indexing.carbon

```diff
@@ -6,11 +6,7 @@
 
 var a: (i32, i32) = (12, 6);
 // TODO: Only diagnose this once, and make the diagnostic clearer.
```

Comment:
This TODO no longer applies here. (The underlying issue hasn't been addressed, but we have a `context.TODO` call to track that.)
```suggestion
```

Reply:
Done